### ISSUE TITLE
Add link to Vercel in the footer

### ISSUE THREAD

**Issue by flybayer** [blitz-js/blitzjs.com#49]
Vercel is sponsoring our account for free hosting! So we need to add a link to them in our footer.

- Text: "Hosted on Vercel"
- Url: `https://vercel.com/?utm_source=blitzjs`

**Comment by simonpeterdebbarma** [comment]
Gotcha. Taking this issue.

**Comment by simonpeterdebbarma** [comment]
The current Footer content comes from `docusaurus.config.js`. 
Only a few properties are possible in that document, these include: style, links, logo,and copyright. 

I found out about swizzling. It's similar to shadowing in Gatsby.
https://v2.docusaurus.io/docs/using-themes/#swizzling-theme-components

`yarn run swizzle @docusaurus/theme-classic Footer`

Used that to create a custom Footer component, which still takes information from the config file. Then, added the Vercel link in `theme/Footer`.

I had one issue. I wanted to change the default color for links, but could not add custom styles by inserting styles in the generated `styles.module.css`. So I created a `footer_styles.css` file in the Footer folder and imported it.
